src/content/glossary.tsx

```diff
@@ -1,17 +1,17 @@
 export const terms = {
     affinity: {
         definition:
-            "Measure of the strength of an interaction between two molecules. Generally, the more chemical bonds that form between two molecules, the stronger the affinity of those molecules for each other.",
+            "The strength of an interaction between two molecules. Two molecules that have high affinity stay bound to each other for longer than two molecules that have low affinity for each other.",
```

Comment:
I updated a few of these definitions

Reply:
Is it also true that two molecules that have high affinity are more likely to become bound?

Reply:
yes in that it is a ratio, ie the forward reaction verses the reverse reaction. but overwhelming the thing that effects binding affinity is the off rate. In general things have an equal chance of bumping into each other